### PR TITLE
FCR page Icon Draw fix

### PR DESCRIPTION
**Pull request by BradMick**
Adds a 'blank' icon to FzaIconTypes and corrects drawing logic on the FCR page.

### CODE REVIEW

**Review by Ollieollieolllie — DISMISSED** [review]
Looks fine, tested it with no issuies

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
If we don't want something to be shown, could we instead not add an icon to be rendered?

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFCRDraw.sqf [hunk 1]
```diff
@@ -151,30 +151,33 @@ private _pointsArray = [];
...
+            continue;
+        };
     };
     //Unit select status
-    if (_forEachIndex == _ntsIndex) then {
-        if (_wasState == WAS_WEAPON_NONE) then {
-            _unitSelAndWpnStatus = ["NTS", "NOMSL"];
-        } else {
-            _unitSelAndWpnStatus = ["NTS"];
+    if(_unitStatus !=  "BLANK") then {
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFCRDraw.sqf:165]: Could `_unitStatus` ever be blank now? I think this if statement is now redundant

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/FzaIconTypes.hpp [hunk 2]
```diff
@@ -608,6 +608,20 @@ class FzaIconTypes {
...
+        tex1 = ""; 
+        tex2 = "";
+        type  = POINT_TYPE_FCR;
+        color = MPD_ICON_COLOR_YELLOW;
+    };
+    class FCR_HELI_BLANK:FCR_ADU_BLANK {};
+    class FCR_TRACK_BLANK:FCR_ADU_BLANK {};
+    class FCR_UNK_BLANK:FCR_ADU_BLANK {};
+    class FCR_WHEEL_BLANK:FCR_ADU_BLANK {};
+    class FCR_FLYER_BLANK:FCR_ADU_BLANK {};
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/FzaIconTypes.hpp:623]: We no longer need these now right?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/FzaIconTypes.hpp (hunk 2 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/FzaIconTypes.hpp:623]: Deleted...we shouldn't

**Review by mattysmith22 — APPROVED** [review]
Nice small PR, good job!

**Pull request merged**